.golangci.yml

```diff
@@ -30,6 +30,8 @@ linters-settings:
   gofmt:
     # simplify code: gofmt with `-s` option, true by default
     simplify: true
+  dupl:
+    threshold: 250
```

Comment:
Shouldn't a local `nolint` exception be added instead?

Reply:
didn't realize you could nolint a whole function, fixed